### PR TITLE
Fix diagnostics

### PR DESCRIPTION
**Pull request by ildyria**
More diagnostics. :)
Should help with #1149

### CODE REVIEW

File: app/Actions/Diagnostics/Checks/IniSettingsCheck.php [hunk 1]
```diff
@@ -51,9 +48,8 @@ public function check(array &$errors): void
 			$errors[]
 				= 'Warning: You may experience problems when uploading a photos of large size. Take a look in the FAQ for details.';
 		}
-		if (
-			intval(ini_get('max_execution_time')) < 200
-		) {
+		$max_execution_time = intval(ini_get('max_execution_time'));
+		if (0 < $max_execution_time && $max_execution_time < 200) {
 			$errors[]
 				= 'Warning: You may experience problems when uploading a large amount of photos. Take a look in the FAQ for details.';
```
Comment (kamil4) [inline app/Actions/Diagnostics/Checks/IniSettingsCheck.php]: I know this isn't one of the lines you changed, but I have a problem with it. Basically, the diagnostics is bogus. The value of `max_execution_time` is irrelevant to the success of uploading multiple photos. Photos are uploaded one-per-API-call. It used to matter for _import_, but these days we manually reset it for every imported photo so it's irrelevant there as well.

Basically, as far as photos are concerned, the value needs to be set to something long enough to process a _single_ photo. I would imagine that for any remotely sensible server, `30` would be enough, though I guess for some low-end RPi more may be needed?

The limit is more likely to be hit if somebody has thousands of albums than during photo upload/import...

Comment (ildyria) [reply app/Actions/Diagnostics/Checks/IniSettingsCheck.php]: > 30 would be enough, though I guess for some low-end RPi more may be needed?

Or... very very big pictures... :see_no_evil:

File: app/Actions/Diagnostics/Checks/IniSettingsCheck.php [hunk 2]
```diff
@@ -77,5 +73,10 @@ public function check(array &$errors): void
 			$errors[]
 				= 'Warning: exec function has been disabled. You may experience some error 500, please report them to us.';
 		}
+
+		if (empty(ini_get('user_agent'))) {
+			$errors[]
+				= 'Warning: php_agent is not set. You may experiences problems when importing images via URL.';
```
Comment (nagmat84) [inline app/Actions/Diagnostics/Checks/IniSettingsCheck.php]: Proposal for better message: "Warning: user_agent for PHP is not set. .....".
The configuration option `user_agent` should be named correctly.

**Review by nagmat84 — CHANGES_REQUESTED** [review]
LGTM. One minor proposal. I have no opinion about the discussion which time limit suffices as execution time limit, so I am out.

File: app/Actions/Diagnostics/Checks/IniSettingsCheck.php (hunk 2 above)
Comment (ildyria) [reply app/Actions/Diagnostics/Checks/IniSettingsCheck.php]:
```suggestion
				= 'Warning: php_agent for PHP is not set. You may experiences problems when importing images via URL.';
```

Comment (ildyria) [reply app/Actions/Diagnostics/Checks/IniSettingsCheck.php]:
```
```suggestion
```
is the way to go @nagmat84  ;)

Comment (nagmat84) [reply app/Actions/Diagnostics/Checks/IniSettingsCheck.php]:
```suggestion
				= 'Warning: user_agent for PHP is not set. You may experiences problems when importing images via URL.';
```

I added my counter-suggestion ;-) I wanted to say that the name of the config option is `user_agent` not `php_agent`.

File: app/Actions/Diagnostics/Checks/IniSettingsCheck.php [hunk 3]
```diff
@@ -77,5 +73,10 @@ public function check(array &$errors): void
 			$errors[]
 				= 'Warning: exec function has been disabled. You may experience some error 500, please report them to us.';
 		}
+
+		if (empty(ini_get('user_agent'))) {
+			$errors[]
+				= 'Warning: user_agent for PHP is not set. You may experiences problems when importing images via URL.';
```
Comment (d7415) [inline app/Actions/Diagnostics/Checks/IniSettingsCheck.php]:
```suggestion
				= 'Warning: user_agent for PHP is not set. You may experience problems when importing images via URL.';
```

**Review by nagmat84 — APPROVED** [review]

File: app/Actions/Diagnostics/Checks/IniSettingsCheck.php (hunk 1 above)
Comment (nagmat84) [reply app/Actions/Diagnostics/Checks/IniSettingsCheck.php]: Is this conversation resolved? Can we merge?

**Review by nagmat84 — APPROVED** [review]

**Review by kamil4 — APPROVED** [review]

**Pull request merged**